One-Foot-In/ofi-alumni
# Add Landing Page and move Login to a modal

This PR introduces a landing page when the user does not have a login session. This page serves as a pitch to any potential member to sign up.

Log In button when visible should open the Login Form in a modal to allow for maximum screen real estate for the landing page.

_Screenshots_
This branch is deployed to test environment as the screenshots would be too many to share so please check them out [here](https://test.onefootin.org).

_Test steps_
1. Navigate to OFI
2. You should see 4 tabs worth of content for landing pages: About Us, Join Us, Our Story, FAQs
